packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VisXZoom/VisXZoom.js

```diff
@@ -123,11 +123,15 @@ function VisXZoom({
     zoomRef.current.dragEnd()
   }
 
+  function wheelDelta(event) {
+    const { zoomInValue, zoomOutValue } = zoomConfiguration
+    const zoomValue = (-event.deltaY > 0) ? zoomInValue : zoomOutValue
```

Comment:
I tried tuning these in #6757, and you can get a much smoother wheel zoom with a smaller delta, like:
```js
    const zoomValue = (-event.deltaY > 0) ? 1.1 : 0.9
```

NB: scale factors of 1.2 (zoom in) and 0.8 (zoom out) are baked into the tests for `VisXZoom` and `ZoomingScatterPlot`, so changing the wheel delta also means changing a few tests. See #6757 for details.